web-components/src/components/modal/TxModal.tsx

```diff
@@ -53,6 +56,88 @@ export interface TxModalProps extends RegenModalProps {
   title?: string;
 }
 
+interface CardItemValueProps {
+  value: ItemValue;
+  color?: string;
+  linkComponent: LinkComponentProp;
+}
+
+const CardItemValue = ({
```

Comment:
Should we put these new components in their own the simplify TxModal component?
ie: `TxModal.CardItemValue` and `TxModal.CardItemValueList`

Reply:
yes, it seems good to me! And what do you think about directly extracting the `TxModal.CardItem` component? (perhaps even encapsulating the two subcomponents for the Value part of the CardItem?

Reply:
I see that it involves many changes in dependencies with other components (in many cases due to the Item interface). I'm going to keep it simpler by extracting the new subcomponents only as you suggest.